pkg/controller/provider/web/ovirt/workload.go

```diff
@@ -262,6 +264,15 @@ func (r *Workload) Expand(db libmodel.DB) (err error) {
 		return err
 	}
 	r.DataCenter.With(dataCenter)
+	// Server CPU
+	serverCpu := &model.ServerCpu{
+		Base: model.Base{ID: "ServerCPUList"},
```

Comment:
how about making `ServerCPUList` a const?
and can't we leverage the cluster we queried here to get only the relevant CPUs for the cluster rather than those for all the cluster levels?

Reply:
hmm, maybe if we change they way we set it in our DB. currently it's taken "as is" from ovirt.

Reply:
ok, so let's check the possibility of changing that please